config/dev.exs

```diff
@@ -66,50 +66,31 @@ config :oli, :cashnet_provider,
   cashnet_client: System.get_env("CASHNET_CLIENT"),
   cashnet_gl_number: System.get_env("CASHNET_GL_NUMBER")
 
-# Configurable http/https protocol options for cowboy
-# https://ninenines.eu/docs/en/cowboy/2.5/manual/cowboy_http/
-http_max_header_name_length =
-  System.get_env("HTTP_MAX_HEADER_NAME_LENGTH", "64") |> String.to_integer()
-
-http_max_header_value_length =
-  System.get_env("HTTP_MAX_HEADER_VALUE_LENGTH", "4096") |> String.to_integer()
-
-http_max_headers = System.get_env("HTTP_MAX_HEADERS", "100") |> String.to_integer()
-
 # For development, we disable any cache and enable
 # debugging and code reloading.
 #
 # The watchers configuration can be used to run external
-# watchers to your application. For example, we use it
-# with webpack to recompile .js and .css sources.
+# watchers to your application. For example, we can use it
+# to bundle .js and .css sources.
 config :oli, OliWeb.Endpoint,
   http: [
-    port: String.to_integer(System.get_env("HTTP_PORT", "80")),
-    protocol_options: [
-      max_header_name_length: http_max_header_name_length,
-      max_header_value_length: http_max_header_value_length,
-      max_headers: http_max_headers
-    ]
+    port: String.to_integer(System.get_env("HTTP_PORT", "80"))
   ],
   url: [
-    scheme: System.get_env("SCHEME", "https"),
+    scheme: System.get_env("SCHEME", "http"),
```

Comment:
These are som config changes/cleanup to help facilitate development mode using HTTP, which is required for the demo tool since it fails the handshake with an invalid SSL cert